src/MeetingsSDKAdapter.js

```diff
@@ -943,15 +942,29 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         meetingWithMediaShareEvent$,
         meetingWithMediaStoppedShareEvent$,
         meetingWithRosterToggleEvent$,
-      ).pipe(map(() => this.meetings[ID])); // Return a meeting object from event
+      ).pipe(map(() => {
+        const {remoteVideo, state} = this.meetings[ID];
+
+        if (state === MeetingState.ENDED) {
+          return this.meetings[ID];
+        }
+
+        if (remoteVideo !== null) {
+          this.meetings[ID].state = MeetingState.ACTIVE;
+        } else {
+          this.meetings[ID].state = MeetingState.JOINED;
```

Comment:
@lalli-flores it seems to me that JOINED is actually the same as ACTIVE, I mean, after you join, you get a different set of meeting controls (eg: Leave meeting). I feel we need to have another talk an these, maybe align them with (or choose a subset of) the sdk meeting states https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/plugin-meetings/src/constants.js#L795